src/shared/components/TitleArea/TitleArea.style.tsx

```diff
@@ -32,7 +28,7 @@ export const StyledInput = styled.input<StyledInputProps>`
 
   line-height: 1;
   padding: ${sizes(1)} 0 ${sizes(2)} ${sizes(2)};
-  ${fluidRange({ prop: 'fontSize', fromSize: '32px', toSize: '40px' })};
+  ${fluidRange({ prop: 'fontSize', fromSize: '32px', toSize: '48px' })};
```

Comment:
I don't think this matches the design anymore. The design specifies `large` and `small` variants for this component, let's follow that.